src/Selection.js

```diff
@@ -9,8 +9,9 @@ import equals from 'fast-deep-equal';
  */
 export default class Selection {
 
-  constructor(target, body) {
+  constructor(creator, target, body) {
     this.underlying = {
+      creator,
```

Comment:
This interface change is made because of https://github.com/Pathlight/recogito-js/pull/4